test_data/asm/keccak.asm

```diff
@@ -0,0 +1,200 @@
+use std::array;
+use std::utils;
+
+// function definition
+let or: int, int -> int = |a, b| a | b;
+let and: int, int -> int = |a, b| a & b;
+let xor: int, int -> int = |a, b| a ^ b;
+let not: int -> int = |a| a ^ 0xffffffffffffffff; // bitwise not for 64 bits
+
+// ln 9 - 12
+// TODO: check that these are good for word size 32
+let RHO: int[] = [
+    1,  3,  6,  10, 15, 21, 28, 36, 45, 55, 2,  14,
+    27, 41, 56, 8,  25, 43, 62, 18, 39, 61, 20, 44
+];
+
+// ln 14 - 16
+// TODO: check that these are good for word size 32
+let PI: int[] = [
+    10, 7,  11, 17, 18, 3, 5,  16, 8,  21, 24, 4,
+    15, 23, 19, 13, 12, 2, 20, 14, 22, 9,  6,  1
+];
+
+// inverse of PI using 1 index plus zero at the beginning
```

Comment:
Please also add a more formal statement about the relation between PI and PI_INVERSE

Reply:
Done